pkg/jsonpath/v1alpha1/json_path.go

```diff
@@ -20,10 +20,11 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	. "github.com/openebs/maya/pkg/msg/v1alpha1"
+	"reflect"
+
+	_ "github.com/openebs/maya/pkg/msg/v1alpha1"
```

Comment:
Please do  not use underscore import, it is mainly use for import side effects.
Use alias e.g. `pkg_msg_v1alpha1` : 
pkg_msg_v1alpha1 "github.com/openebs/maya/pkg/msg/v1alpha1"
You cannot use simply `v1alpha1` here as it will say redundant aliasing.
cc @princerachit @Akash4927